ui/src/views/helpers/index.ts

```diff
@@ -308,7 +308,7 @@ const NEW_VIEW_CREATORS = {
           },
         },
       ],
-      colors: NINETEEN_EIGHTY_FOUR,
```

Comment:
Switching over to a different default allows for a consistent color schema for the CheckViewProperties. This ensures that both the view history and default view are the same type